packages/azure_openai/changelog.yml

```
@@ -1,7 +1,12 @@
 # newer versions go on top
+- version: "0.5.0"
```

```suggestion
- version: "0.4.1"
```
Since its a bug fix.